control-plane-operator/controllers/hostedcontrolplane/kas/deployment.go

```diff
@@ -967,3 +966,12 @@ func buildKonnectivityVolumeClusterCerts(v *corev1.Volume) {
 		DefaultMode: pointer.Int32(0640),
 	}
 }
+
+func renderAuditLogScript(auditLogFilePath string) string {
+	var script = `#!/bin/bash
+trap exit SIGTERM
```

Comment:
Are you relying on implicit behavior of `bash` when you call `exit`? Could we instead explicitly forward `SIGTERM` to children on `SIGINT`?

Reply:
@stevekuznetsov is this what you'd like Joe to do: https://unix.stackexchange.com/questions/146756/forward-sigterm-to-child-in-bash?

Reply:
```sh
#!/usr/bin/env bash

set -o errexit
set -o nounset
set -o pipefail

function cleanup() {
  for child in $( jobs -p ); do
    kill "${child}"
  done
  wait
}
trap cleanup EXIT

for (( i = 0; i < 999; i++ )); do
    child.sh $i
done
```

Reply:
@Joseph-Goergen ptal